data/human/human missions.txt

```diff
@@ -7497,3 +7497,211 @@ mission "Adelita 5"
 			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`
 			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
 			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`
+
+
+
+mission "Syndicate Business 1"
+	name "Rescue kidnapped man in <waypoints>"
+	description "You have received a mysterious distress call from an alleged abductee in the <waypoints> system. Head there to see if you can find the signal's source."
+	minor
+	to offer
+		random < 40
+		credits > 5000000
+		"combat rating" > 50
+	source
+		government "Syndicate"
+		attributes "core"
+	waypoint "Moktar"
+	destination "Sunracer"
+	on offer
+		conversation
+			`As you are walking through the spaceport, you are startled by a sudden loud beeping from a nearby computer terminal. The computer is unattended and the screen is blinking on and off.`
+			choice
+				`	(Check it out.)`
+				`	(Ignore it.)`
+					decline
+			`	When you approach, you are greeted by plain text on the terminal's screen:`
+			`	"SOS: Kidnapped by pirates, can't respond. <waypoints> standard coordinates: RAD AU 30.12 LAT 0.2 LON 303.9. Life in danger, need urgent rescue. Cannot communicate, all signals monitored. Please help!"`
+			`	You note that the message appears to be coming from an emergency transmitter commonly found on many small shuttles and escape pods. There is no shortage of pirates in <waypoints>.`
```

Comment:
Thanks for the feedback! Having a manual vs automated message is very much a gameplay-over-realism choice. Beyond being a bit more colorful than "SOS: < waypoints> : RAD AU 30.12 LAT 0.2 LON 303.9," the message establishes:

1. **Howard's panic/desperation**: Howard is panicking and not thinking rationally. The fact that he's sending a manual message instead of an automated one also reinforces that a bit, and making his situation maybe a tad more personal. 
2. **Pirates**: Very clearly setting the stage for what dangers to expect (expect violence rather than a more relaxed mechanical malfunction)

Both of which would be lost with the automated message. 

That being said, **mOctave** also thought the original version of this message was odd (though it's been reworked since), so it may make sense to change to the simplified one. 

I guess give a thumbs up if this comment addressed your concerns, or a side eye if it does not and I'll swap over to the automated message.